src/spyglass/common/common_ephys.py

```diff
@@ -293,7 +293,12 @@ class Raw(SpyglassIngestion, dj.Imported):
 
     _nwb_table = Nwbfile
     _only_ingest_first = True
-    _source_nwb_object_name = "e-series"
+    _source_nwb_object_name = [
```

Comment:
Ideally we'd defer to nwb naming conventions as the source of truth, but at least the names are transparently associated with the table in one place. I just worry about the feasibility of us doing fix PRs like this whenever requested. The alternative is making a user-managed config file somewhere to handle this variability, but that seems equally messy